x-pack/plugins/lens/public/visualizations/xy/color_assignment.ts

```diff
@@ -125,7 +125,7 @@ export function getAssignedColorConfig(
     const annotation = layer.annotations.find((a) => a.id === accessor);
     return {
       columnId: accessor,
-      triggerIcon: annotation?.isHidden ? ('invisible' as const) : ('color' as const),
+      triggerIconType: annotation?.isHidden ? ('invisible' as const) : ('color' as const),
```

Comment:
no need for type assertion here

Reply:
Fixed [235e89e](https://github.com/elastic/kibana/pull/147686/commits/235e89ec86bda109bca48c952192c1cf93ab0b44)